cost/google/schedule_instance/README.md

```diff
@@ -0,0 +1,78 @@
+# Google Schedule Instance
+
+## What it does
+
+This Policy Template allows you to schedule start and stop times for your instance, along with the option to terminate instance, update and delete schedule.         
+
+## How to Use
+
+This policy relies on a label with format 'schedule' to stop and start instances based on a schedule. The label value defines the schedule with a start time(start hour and start minute), stop time(stop hour and stop minute), days of the week and timezone. The start and stop time are in 24 hour format, and the days of the week are two character abbreviation for example: mo, tu, we. See full example below.. Use a Timezone TZ value to indicate a timezone to stop/start the instance(s) 
+
+## schedule Label Example
+
+Since label supports only `-`, `_`, lowercase characters, numbers and International characters, We are replacing `:` with `m` in start and stop time, and ASCII code for special character `/` and `+` in timezone, surrounded by `_`.
```

Comment:
I'm not excited about the use of ascii code for special characters.   See notes in JIRA.